editor/editor_node.cpp

```diff
@@ -2368,6 +2368,7 @@ void EditorNode::push_item(Object *p_object, const String &p_property, bool p_in
 	}
 	_add_to_history(p_object, p_property, p_inspector_only);
 	_edit_current();
+	editor_history.cleanup_history();
```

Comment:
I'm not sure if I should call this only in _edit_current() only for MultiNodeEdit, which would make sense to avoid unnecessary cleanups, but maybe it is needed for more things I'm not even aware of ?